### DISCUSSION TITLE
DDS Alpha format (8bpp-A8)

### DISCUSSION THREAD

**Discussion by valllex** [ImageMagick/ImageMagick#6702] [Contribute]
Is there any chance for DDS Alpha (8bpp-A8) support? 
After all the format isn't very difficult! Only one channel - 8-bit uncompressed data! :)
A8 sample image: http://divastudio.net/val/A8sampe.dds
PS: I'll be happy just for A8 input for now (read only)! Thanks!

**Comment by dlemstra** [comment]
[A8sampe.zip](https://github.com/ImageMagick/ImageMagick/files/961560/A8sampe.zip)
